apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -7,7 +7,7 @@ import { useLocale } from "@calcom/lib/hooks/useLocale";
 import { TimeFormat } from "@calcom/lib/timeFormat";
 import { nameOfDay } from "@calcom/lib/weekday";
 import type { Slot } from "@calcom/trpc/server/routers/viewer/slots";
-import { SkeletonContainer, SkeletonText } from "@calcom/ui";
+import { SkeletonContainer, SkeletonText } from "@calcom/ui/v2";
```

Comment:
Would it make sense to get rid of v1 Skeletons altogether?

Reply:
Yes, asap indeed. Eg console is still using it, so I'm working hard om removing them from there as well, so I can delete all of those v1 components. 🙏